/tmp/flask-restx/flask_restx/inputs.py:36:16: DUO138 catastrophic "re" usage - denial-of-service possible
/tmp/flask-restx/flask_restx/inputs.py:50:15: DUO138 catastrophic "re" usage - denial-of-service possible
